mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -447,9 +553,50 @@ public void onResume() {
     private void resetStateFragmentAfterResume(FragmentManager fm) {
         stateFragment = new StateRetainFragment();
         fm.beginTransaction().add(stateFragment, "stateFragment").commit();
-        mOpenHABTracker = new OpenHABTracker(this, openHABServiceType);
         mStartedWithNetworkConnectivityInfo = NetworkConnectivityInfo.currentNetworkConnectivityInfo(this);
-        mOpenHABTracker.start();
+
+        onConnectivityChanged();
+        try {
+            initializeConnectivity();
+        } catch (ConnectionException e) {
+            // will be handled by #getConnection() later
+        }
+    }
+
+    @Override
+    protected void onEnterNoNetwork() {
+        super.onEnterNoNetwork();
+        ViewPager pager = findViewById(R.id.pager);
+        if (pager != null) {
+            pager.removeAllViews();
+        }
+
+        if (mDrawerLayout != null) {
+            mDrawerLayout.closeDrawers();
+            mDrawerLayout.setDrawerLockMode(DrawerLayout.LOCK_MODE_LOCKED_CLOSED);
+        }
```

Comment:
This will essentially also lock down access to settings. Is this intended?

Reply:
Why would one want to open the settings in that case? We could, however, think about having a drawer without sitemaps?

Reply:
Yeah, that's (drawer without sitemaps) what I was thinking as well. About settings, dunno, but preventing drawer access completely feels a bit heavy handed.

Reply:
(See my current work branch at https://github.com/maniac103/openhab.android/tree/cleanup-fragment-management-after-pr562 for where I'm heading at UI wise)